objectivec/Tests/GPBMessageTests+ClassNames.m

```diff
@@ -0,0 +1,164 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Since this is new, you will need to add it to `Makefile.am` with the others

Reply:
Done